sdk/metricsadvisor/ai-metrics-advisor/src/metricsAdvisorClient.ts

```diff
@@ -516,63 +518,11 @@ export class MetricsAdvisorClient {
     }
   }
 
-  /**
```

Comment:
Where are we using the private method?

Reply:
Inside the implementation of the `listAnomalies()` overloads